multicluster/Makefile

```diff
@@ -50,6 +50,12 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
+.PHONY: test
+test:  
+	$(CURDIR)/hack/verify_kind.sh 
+	$(CURDIR)/ci/test_integration.sh
```

Comment:
move test_integration.sh to `test/integration` directory, unnecessary to add a `ci` folder.